src/framework/theme/components/layout/layout.component.ts

```diff
@@ -291,14 +292,16 @@ export class NbLayoutComponent implements AfterViewInit, OnInit, OnDestroy {
     protected elementRef: ElementRef,
     protected renderer: Renderer2,
     protected router: Router,
+    @Inject(PLATFORM_ID) protected platformId: Object,
+    @Inject(DOCUMENT) protected document: any,
```

Comment:
So this is provided by angular, right? Shouldn't we provide this manually?

Reply:
Yep, `PLATFORM_ID` and `DOCUMENT` are provided by angular.